include/openmc/mesh.h

```diff
@@ -453,7 +461,12 @@ class UnstructuredMesh : public Mesh {
   static const std::string mesh_type;
   virtual std::string get_mesh_type() const override;
 
+  //! Sample a tetrahedron for an unstructured mesh
```

Comment:
I believe since I'm calling the method from a derived class it must be protected rather than private. That is what will be implemented in the next commit